### ISSUE TITLE
Unable to run newer versions of Elementum add-on past version 0.1.99

### ISSUE THREAD

**Issue by alv32** [elgatito/plugin.video.elementum#1072]
For reason unknown, my device can only work with version 0.1.99.

## Expected Behavior

Updated Elementum add-on should run normally.

## Current Behavior

Elementum add-on crashed a few times when starting the daemon and eventually aborts.

## Steps to Reproduce (for bugs)

1. Update Elementum add-on to any version newer than 0.1.99.
2. After the successful update, Elementum will attempt to start the daemon.
3. Elementum will crash a few times and finally abort.

## Context

I realized that newer versions will all crash if the the add-on folder exist in addon_data. Removing this folder will allow Elementum to start normally at the expense of customized settings, Trakt login, etc.

## Your Environment

* Version used: 0.1.103 & 0.1.105

* Environment name and version: Kodi 21.1 for Android

* Operating System and version: Android 14

* Link to Kodi debug log:

1. Screenshot of the content in the Elementum add-on folder under addon_data.
![addon_data](https://github.com/user-attachments/assets/1b13ac62-1b73-47bf-8ad0-91aaf6b5e34b)

2. Kodi log when Elementum add-on crashed a few time.
[kodi-failed.log](https://github.com/user-attachments/files/17302024/kodi-failed.log)

3. Elementum log when Elementum add-on crashed a few time.
[elementum.log](https://github.com/user-attachments/files/17302026/elementum.log)

4. Kodi log when Elementum add-on ran normally after removing the Elementum add-on folder under addon_data.
[kodi-success.log](https://github.com/user-attachments/files/17285838/kodi-success.log)

**Comment by elgatito** [comment]
@alv32 Strange there were no reports like that before.

I think this is the thing you observe: https://github.com/etcd-io/bbolt/issues/558

And the fix for that issue is included in the version that requires newer Go version.
And we were not updating that version, because it would require Windows 10 as the minimal version for Elementum. 
Not sure what are the numbers for Windows 7 and lower, and how many users use Elementum there.

**Comment by elgatito** [comment]
@alv32 Can you please try to use version 0.1.106.

**Comment by alv32** [comment]
Thanks elgatito.

I am going to give 0.1.106 a go later today and report my findings.

**Comment by alv32** [comment]
@elgatito I can confirm version 0.1.106 resolved the reported issue.

I rolled back my setup which is running version 0.1.99 and burst 0.0.89 then proceed to update both add-ons via the Elementum repo. Once Elementum add-on is updated, the daemon started successfully. At this point, I exited Kodi and start it again to ensure the Elementum daemon start successfully for a second time.

- Link to Kodi debug log:

1. Kodi log after completing the updates from previous versions.
[kodi.old.log](https://github.com/user-attachments/files/17320186/kodi.old.log)

2. Screenshot of the content in the Elementum add-on folder under addon_data.
![addon_data](https://github.com/user-attachments/assets/5ac708ab-f16c-44d4-9912-66b45ac77d4c)

3. Elementum log after I have exited and reenter Kodi to ensure Elementum daemon start.
[elementum.log](https://github.com/user-attachments/files/17320187/elementum.log)

4. Kodi log from the second Kodi instance after updating both add-ons.
[kodi.log](https://github.com/user-attachments/files/17320188/kodi.log)

Thank you so much for sorting this out 👍
